app/src/main/java/com/github/globant/githubsubscribers/commons/utils/Constants.java

```diff
@@ -12,6 +12,7 @@
 
     public static final String DEBUG_PREFIX = "GITHUBAPI";
     public static final boolean DEBUG_LOGS = true;
+    public final static String MESSAGE_FAILED_SERVICE = "The sevice is not available";
```

Comment:
This part will be to get better by Edwin, he will implement a best way for manage the errors in the next PR, because he started this task